### PR TITLE
Ft enrollment for user

### PR DESCRIPTION
**Pull request by RobertOdhiz**

### CODE REVIEW

File: src/services/enrolloments.services.js [hunk 1]
```diff
@@ -24,19 +25,34 @@ export class EnrollmentService {
     return await Enrollment.create(newEnrollment);
   }
 
+  static async getEnrollmentsByUser (userId) {
+    return await Enrollment.findAll({ where: { userId }, 
+    include: [
+      {
+        model: Project,
+        as: 'project',
+        attributes: ['id', 'title']
```
Comment (fabmukunzi) [inline src/services/enrolloments.services.js:34]: Let's include all projects data

File: src/services/enrolloments.services.js [hunk 2]
```diff
@@ -24,19 +25,34 @@ export class EnrollmentService {
...
+
   static async getSingleEnrollment (id) {
     return await Enrollment.findOne({ 
       where : { id },
       include: [
         {
           model: User,
           as: 'enrolledUser',
-          attributes: ['firstName', 'lastName']
+          attributes: ['id', 'firstName', 'lastName', 'email']
```
Comment (fabmukunzi) [inline src/services/enrolloments.services.js:46]: add more info like profileImage

**Review by fabmukunzi — CHANGES_REQUESTED** [review]

File: src/services/enrolloments.services.js (hunk 1 above)
Comment (RobertOdhiz) [reply src/services/enrolloments.services.js:34]: Have a look at this in the new branch [rf-enrollment here](https://github.com/fabmukunzi/posinnove-be/pull/29)

Comment (RobertOdhiz) [reply src/services/enrolloments.services.js:34]: Once [this branch](https://github.com/fabmukunzi/posinnove-be/pull/29) is merged with dev. Then I can resolve the conflicts and then ensure proper data is returned as from the requested changes.

**Pull request closed without merging**